src/mmw/js/src/core/views.js

```diff
@@ -2,7 +2,7 @@
 
 var $ = require('jquery'),
     L = require('leaflet'),
-    _ = require('lodash'),
+    _ = require('underscore'),
```

Comment:
Yeah, we should drop one.  I'm a little partial to the expanded `lodash` methods, and if there is a version compatible with backbone, I'd be happy to use that.  Aside from that, they're both solid libraries and I haven't got a strong opinion.  I suspect the size difference between the two is negligible, so I think it comes down to backbone support.  If lodash brings in a ton of dependencies, that would also be a strike against it for me.